### PR TITLE
update(handler): add 'shouldReport' function

### PR DESCRIPTION
**Pull request by choyri**
Maybe, someone will use `app(ExceptionHandler::class)->shouldReport($e)` .

### CODE REVIEW

File: src/NewRelicExceptionHandler.php [hunk 1]
```diff
@@ -77,4 +77,13 @@ protected function logException(Exception $e)
...
     }
 
+    /**
+     * Original handler's function
+     *
+     * @param Exception $e
+     */
+    public function shouldReport(Exception $e)
+    {
+        return $this->primaryHandler->shouldReport($e);
```
Comment (Jalle19) [inline src/NewRelicExceptionHandler.php:87]: There is no property named primaryHandler so not sure how this is supposed to work?

Comment (choyri) [reply src/NewRelicExceptionHandler.php:87]: 🤔 This is really a problem, although in README the default primaryHandler is `App\Exceptions\Handler`.

Comment (Jalle19) [reply src/NewRelicExceptionHandler.php:87]: You can create a PR against https://github.com/digiaonline/lumen-chained-exception-handler, that's what's used here

**Comment by Jalle19** [comment]
This PR got closed because I changed the default branch to `master`, but as mentioned in the comments the change you're looking for should be done in https://github.com/digiaonline/lumen-chained-exception-handler

**Pull request closed without merging**